MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -631,6 +628,7 @@
 				C52AC1251DF0E48600396B9F /* RouteProgressTests.swift */,
 				C5ADFBD91DDCC7840011824B /* Info.plist */,
 			);
+			name = MapboxCoreNavigationTests;
 			path = MapboxNavigationTests;
```

Comment:
This is probably the cause of the test failures: the paths associated with the test bundles’ groups are out of date.